src/Toggle/MysqlActivatedToggleGateway.php

```diff
@@ -43,20 +62,21 @@ public function __construct( Connection $connection, IsToggleActive $toggleCheck
      */
     public function getAllMyActivatedToggles( $userIdentifier )
```

Comment:
The name of this method suggests the gateway knows too much about toggles and a user.  Probably just the name used though.